docs/sdk/quickstart-by-language/quickstart-nodejs.mdx

```diff
@@ -0,0 +1,171 @@
+title: Permit NodeJS SDK
+---
+
+Working with the Permit.io [Insert Language Here] SDK enables developers to seamlessly integrate robust authorization management into
+their applications. This SDK simplifies the implementation of fine-grained access controls, whether you're managing user roles with
+[RBAC](/how-to/build-policies/rbac/overview), policies with [ABAC](/how-to/build-policies/abac/overview), relationships with
+[ReBAC](/how-to/build-policies/rebac/overview), or making use of other powerful Permit features like [Elements](/embeddable-uis/overview).
+
+:::note Prerequisites:
+This tutorial is for users who have already set up at least one policy in Permit.io. If you haven't done so yet,
+please refer to [this guide](/quickstart) first.
+:::
+
+## Adding the Permit.io Node SDK to your project
+
+### 1. Install the Permit.io SDK
+
+To start, you need to install the Permit.io SDK in your project. Open your terminal and run the following command:
+
+```sh
+npm install permitio
+```
+
+This will add the Permit.io SDK to your project, allowing you to integrate authorization management seamlessly.
+
+### 2. Import the SDK into your code
+
+After installing the SDK, you need to import it into your application.
+
+```javascript
+import { Permit } from "permitio";
+```
+
+### 3. Create a new Instance of the SDK
+
+Next, create a new Permit object instance of the Permit SDK. You'll need a secret API key, which you can obtain by following the instructions
+provided in the setup section of the Permit.io documentation.
+
+```javascript
+// This first line initializes the SDK and connects your Node.js app
+// to the Permit.io PDP container you've set up in the previous step.
+const permit = new Permit({
+  // your API Key
+  token: "[YOUR_API_KEY]",
+
+  // in production, you might need to change this url to fit your deployment
+  pdp: "http://localhost:7766",
+
+  // if you want the SDK to emit logs, uncomment this:
+  // log: {
+  //   level: "debug",
+  // },
+
+  // The SDK returns false if you get a timeout / network error
+  // if you want it to throw an error instead, and let you handle this, uncomment this:
+  // throwOnError: true,
+});
+```
+
+Replace **`YOUR_API_KEY`** with the actual API key you obtained. Find instructions how to fetch your API
+key [here](/overview/connecting-your-app#1-get-your-permit-environment-api-key).
+
+## The SDK enforcement functionality - Checking for permissions
+
+`permit.check()` is the fundamental SDK function that controls the enforcement in your application level code. It acts as a guard of who
+can do what, based on the policy that you defined.
+
+If you would like to learn more about the `permit.check()` function, and how it works with different policy models, you can read
+more about it [here](/how-to/enforce-permissions/check).
+
+You can run a permission check with permit.check(), passing in 3 arguments:
+
+- `user`: a unique string id that identifies the user performing the action.
```

Comment:
If we are showing the arguments in this document, we should also show the various types of the user/resource argument types. User could be an object, not only a string